### PR TITLE
Add role update transverse node

### PR DESCRIPTION
**Pull request by Thiblef**

### CODE REVIEW

File: Backoffice/AuthorizeEdition/NodeVersionStrategy.php [hunk 1]
```diff
@@ -49,7 +50,11 @@ public function support($document)
      */
     public function isEditable($document)
     {
-        if (!$this->autorizationChecker->isGranted(TreeNodesPanelStrategy::ROLE_ACCESS_UPDATE_NODE)) {
+        if ($document->getNodeType() === NodeInterface::TYPE_DEFAULT && !$this->autorizationChecker->isGranted(TreeNodesPanelStrategy::ROLE_ACCESS_UPDATE_NODE)) {
```
Comment (itkg-alavieille) [inline Backoffice/AuthorizeEdition/NodeVersionStrategy.php]: you can factorize this lines ( 53 - 60)

File: BackofficeBundle/Resources/translations/role.fr.yml [hunk 2]
```diff
@@ -1,5 +1,6 @@
 open_orchestra_role:
     role_access_general_node: Gérer les pages transverses
+    role_access_update_general_node: Editer une page transverse
```
Comment (ggrasset) [inline BackofficeBundle/Resources/translations/role.fr.yml]: Wording : English uses plural, French uses singular.

**Comment by ggrasset** [comment]
:+1: when @itkg-alavieille 's refactoring is done

**Comment by itkg-alavieille** [comment]
you can replace has_role by is_granted also

**Comment by Thiblef** [comment]
@itkg-alavieille @ggrasset

**Comment by itkg-alavieille** [comment]
what happen for listAction if an use is granted for ROLE_ACCESS_UPDATE_GENERAL_NODE but not for ROLE_ACCESS_UPDATE_NODE

**Comment by Thiblef** [comment]
There is no node context in the listBlockAction so I can't check the current node type. 
On the API side, the user has access to the listBlockAction if he is granted for ROLE_ACCESS_UPDATE_GENERAL_NODE or ROLE_ACCESS_UPDATE_NODE.

**Comment by itkg-alavieille** [comment]
:+1:

**Pull request merged**